platform/feature-api/communication/README.md

```diff
@@ -0,0 +1,257 @@
+# Polypoly communication API
+
+## bubblewrap
```

Comment:
I think we should just leave such detailed READMEs in the respective subfolders. Someone using the _communication_ package isn't interested in _bubblewrap_, that's an implementation detail. What the user of the _communication_ package is interested in is the `bootstrap.js` file under _remote-pod_ -> Nothing else.